resources/dicts/thoughts/dead/starclan/general.json

```diff
@@ -224,16 +249,108 @@
+            "Is feeling guilty watching r_c miss {PRONOUN/m_c/object}",
+            "Deeply regrets leaving r_c without {PRONOUN/m_c/object}"
+        ],
+        "random_living_status": [
+            "starclan",
+            "darkforest",
+            "unknownresidence"
+        ],
+        "relationship_constraint": [
+            "parent/child"
+        ],
+        "main_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "dead_parent_to_alive_child2",
+        "thoughts":[
+            "Snuck into r_c's dream to comfort {PRONOUN/r_c/object} after a hard day",
+            "Visited r_c's dream to tell {PRONOUN/r_c/object} just how immensely proud {PRONOUN/m_c/subject} {VERB/m_c/are/is} of {PRONOUN/r_c/subject}"
+        ],
+        "random_living_status": [
+            "starclan",
+            "darkforest",
+            "unknownresidence"
+        ],
+        "main_skill_constrict": [
+            "STAR,1",
+            "DREAM,1"
+        ],
+        "relationship_constraint": [
+            "parent/child"
+        ],
+        "main_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "dead_outsider_parent_to_alive_child",
+        "thoughts":[
+            "Regrets not joining a Clan if it meant being able to raise r_c",
+            "Still doesn't trust Clan cats, but is happy r_c is safe",
+            "Is just glad r_c was saved from meeting the same fate {PRONOUN/m_c/subject} did",
+            "Is so greatful to c_n for saving r_c"
+        ],
+        "random_backstory_constraint": [
+            "orphaned1",
+            "orphaned2",
+            "orphaned4",
+            "orphaned5"
+            ],
+        "main_living_status": [
+            "unknownresidence"
+        ],
+        "relationship_constraint": [
+            "parent/child"
+        ],
+        "random_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "dead_child_to_alive_child_sibling1",
         "thoughts":[
             "Wants to let r_c know {PRONOUN/m_c/subject} {VERB/m_c/are/is} okay",
             "Wishes {PRONOUN/m_c/subject} could play with r_c",
-            "Is watching r_c play"
+            "Is watching r_c play",
+            "Is watching over r_c reminicing over the short time they had together"
+        ],
+        "main_age_constraint": [
+            "newborn",
+            "kitten"
+        ],
+        "relationship_constraint": [
+            "siblings"
+        ],
+        "random_living_status": [
+            "living"
+        ],
+        "random_age_constraint": [
+            "newborn",
+            "kitten"
+        ]
+    },
+     {
+        "id": "dead_child_to_alive_child_sibling2",
+        "thoughts":[
+            "Snuck into r_c's dreams to play with {PRONOUN/r_c/object} one last time",
+            "Visits r_c in {PRONOUN/r_c/poss} to tell them to live for the both of them"
```

Comment:
need something after {PRONOUN/r_c/poss}
reads as 'visits r_c in their to tell them to live...'